library/src/main/java/org/apache/apex/malhar/lib/state/spillable/SpillableByteArrayListMultimapImpl.java

```diff
@@ -152,7 +153,15 @@ public SpillableStateStore getStore()
   @Override
   public List<V> removeAll(@Nullable Object key)
```

Comment:
The map not only exists in the Spillable Map, but could also serialized and copy to the bucket of the state management. and also could be written to one or even multiple files. So I don't think this solution could work.